### PR TITLE
Highlighted prerequisites, new section for PiP and Interpolation services

### PR DESCRIPTION
**Pull request by tigerlily-he**
:wave: I did some awesome work for the Pelias project and would love for everyone to have a look at it and provide feedback.

---
#### Here's the reason for this change :rocket:
I highlighted services/data that were prerequisites for other parts to work (such as who's on first admin hierarchy data for admin lookup). Also, I added links at the top to reference later sections in the page or git repos because the doc is long. 

Included a section on installing point-in-polygon service and interpolation. Those are more services that make search queries return better results.

---
#### Here's what actually got changed :clap:
- [x] Deleted section "Choose your import settings" because admin-lookup is on by default and deduplication is off 
- [x] Linked each step of the installation process to the corresponding page section
- [x] Added section on how to install PiP Service and Interpolation
- [x] Changed Elasticsearch link to go directly to downloading v 2.3.0 rather than ES home page
- [x] Fixed small typos and shortened wording
- [x] Require Libpostal instead of leaving it as optional

---

### CODE REVIEW

**Comment by tigerlily-he** [comment]
Closes https://github.com/pelias/pelias/issues/591

File: INSTALL.md [hunk 1]
```diff
@@ -12,79 +12,63 @@ your setup to a reference. All information on Mapzen Search can be found
...
 
 ## Choose your datasets
 
-Pelias can currently import data from four different sources, using five different importers. The contents and description of these
-sources are available on our [data sources page](https://mapzen.com/documentation/search/data-sources/).
-Here we'll just focus on what to download for each one.
+Pelias can currently import data from [four different sources](https://mapzen.com/documentation/search/data-sources/), using five different importers.
+
+Start with the `Who's on First` dataset. At a minimum, download the admin hierarchy data, required for [providing neighborhood, city, or any region info](https://github.com/pelias/wof-admin-lookup) to imported records (admin lookup).
```
Comment (orangejulius) [inline INSTALL.md]: this is great.

File: INSTALL.md [hunk 2]
```diff
@@ -93,56 +77,8 @@ A full planet PBF file is about 36GB.
 
 #### Street Data (Polylines)
 
-To import street data from OSM, a separate importer is used that operates on a preprocessed dataset
-derived from the OSM planet file. The importer's documentation includes a [download section](https://github.com/pelias/polylines#download-data)
-with instructions on where to get this data.
-
-## Choose your import settings
```
Comment (orangejulius) [inline INSTALL.md]: yesss. so good to see this whole confusing section disappear

File: INSTALL.md [hunk 3]
```diff
@@ -332,22 +270,17 @@ If you're using a terminal, you can also search and/or monitor Elasticsearch usi
...
-install them if not present.
-
 ```bash
 cd schema                      # assuming you have just run the bash snippet to download the repos from earlier
 node scripts/create_index.js
 ```
+The Elasticsearch Schema is analogous to the layout of a table in a traditional relational database,
+like MySQL or PostgreSQL. While Elasticsearch attempts to auto-detect a schema that works when
+inserting new data, this generally leads to non-optimal results. In the case of Pelias, inserting
+data without first applying the Pelias schema will cause all queries to fail completely:
```
Comment (orangejulius) [inline INSTALL.md]: moving this down works great

File: INSTALL.md [hunk 4]
```diff
@@ -406,30 +333,12 @@ data represents a statistical natural language processing model of address parsi
 OpenStreetMap data. The API will also require about 4GB of memory (it used only a few hundred
 before), to store the needed data for queries.
 
-First, install libpostal following its [installation docs](https://github.com/openvenues/libpostal#installation).
-This will also download the training data, so be sure to have enough free disk space.
```
Comment (orangejulius) [inline INSTALL.md]: Hmm. I see you linked to the installation docs for libpostal above, but maybe another link here would be helpful.

Comment (tigerlily-he) [inline INSTALL.md]: Where should it link to, if not the installation docs? just the Github repo?

Comment (orangejulius) [inline INSTALL.md]: No I meant you can keep a link to the installation docs here, to remind people where the directions can be found.

**Comment by dianashk** [comment]
This is very nice! Noticed that we've blown past our previous full planet build size of [340 million records](https://github.com/pelias/pelias/pull/593/files#diff-7d442b7eb49f5fc377f51e74b291cfc1L130) which might be nice to update.

**Comment by tigerlily-he** [comment]
Fixes https://github.com/pelias/pelias/issues/591

**Pull request merged**